server/src/routes.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express'
+import { createUserFactory } from '@modules/CreateUser/CreateUerFactory'
+
+const routes = Router()
+
+routes.post('/users', (request, response) => {
+  return createUserFactory().handle(request, response)
```

Comment:
I suggest name `CreateUserHandler` or `CreateUserController` instead of `createUserFactory` seems more semantic.

Reply:
Do you think it's unnecessary?

Reply:
Adjusted